modules/nextflow/src/main/groovy/nextflow/config/ConfigParser.groovy

```diff
@@ -322,6 +322,8 @@ class ConfigParser {
         LinkedList stack = new LinkedList()
         LinkedList profileStack = new LinkedList()
         stack << [config: config, scope: [:]]
+        boolean intoProfile = false
```

Comment:
here `into` stands for *within* a `profile` block right?

Reply:
yes

Reply:
cool, i'll change to `withinProfile` to make it a bit more explicit